Form-Controls/styles.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Questrial&family=Roboto:wght@100&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  min-height: 100vh;
+  font-family: "Questrial", sans-serif;
+  text-transform: uppercase;
+  background: rgb(224, 224, 231);
+  background: linear-gradient(
+    90deg,
+    rgba(224, 224, 231, 1) 0%,
+    rgba(158, 157, 157, 0.9276304271708683) 52%,
+    rgba(66, 70, 70, 1) 100%
+  );
+  /* font-family: "Roboto", sans-serif; */
+}
+main {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  text-transform: uppercase;
+}
+#form {
```

Comment:
Try to avoid the use of #ids as much as possible in CSS. IDs are extremely powerful; ids outweigh an infinite number of class and element and attribute selectors, so they can be hard to override. Here's a good article digging in to this a bit more : https://css-tricks.com/almanac/selectors/i/id/